Lombiq.HelpfulLibraries.Attributes/Lombiq.HelpfulLibraries.Attributes.csproj

```diff
@@ -2,9 +2,27 @@
 
     <PropertyGroup>
       <TargetFramework>netstandard2.0</TargetFramework>
-      <IsPackable>false</IsPackable>
+      <IsPackable>true</IsPackable>
```

Comment:
Please add the missing NuGet badges for the two new packages to the main readme [here](https://github.com/Lombiq/Helpful-Libraries/blob/dev/Readme.md?plain=1#L3).